x-pack/plugins/observability/public/pages/alert_details/components/header_actions.test.tsx

```diff
@@ -26,12 +27,19 @@ const useKibanaMock = useKibana as jest.Mock;
 const useFetchRuleMock = useFetchRule as jest.Mock;
 const mockCases = casesPluginMock.createStartContract();
 
+const mockHttp = {
+  basePath: {
+    prepend: (url: string) => `wow${url}`,
```

Comment:
Are we testing something with adding `wow` before URL?

Reply:
Not testing particularly prepending the URL, but I needed to mock `http` since I added it [here](https://github.com/elastic/kibana/blob/772c2ae116ee8e0a2e7c36f47b5197407a9bdc1d/x-pack/plugins/observability/public/pages/alert_details/components/header_actions.tsx#L193).